src/promptflow/CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Release History
 
+## 1.6.0 (TBD)
+
+### Features Added
+
+- [SDK/CLI] Support configuring environment variable to directly use `AzureCliCredential` for `pfazure` commands.
```

Comment:
[SDK/CLI] -> [CLI]? as this only appies for CLI

Reply:
well lets change it in the next pr